main_project/test_registration.rb

```diff
@@ -31,7 +33,7 @@ def test_log_out_r
 
     @wait.until {@driver.find_element(:class => 'login').displayed?}
     login_button = @driver.find_element(:class => 'login')
-    assert(login_button.displayed?)
+    expect(login_button.displayed?).to be true
```

Comment:
go through predicate matchers
http://www.relishapp.com/rspec/rspec-expectations/v/3-5/docs/built-in-matchers/predicate-matchers
and try to get rid of simple ```to be true``` matchers